docs/src/content/docs/ref/core-cells.mdx

```diff
@@ -286,6 +290,31 @@ let b: Builder = beginCell();
 let fizz: Builder = b.storeRef(emptyCell());
 ```
 
+## Builder.storeMaybeRef
+
+<Badge text="Available since Tact 1.5" variant="tip" size="medium"/><p/>
+
+```tact
+extends fun storeMaybeRef(self: Builder, cell: Cell?): Builder;
+```
+
+Extension function for the [`Builder{:tact}`][builder].
+
+If the `cell` is not `null{:tact}`, stores $1$ as a single bit and then reference `cell` into the copy of the [`Builder{:tact}`][builder]. Returns that copy.
+
+If the `cell` is `null{:tact}`, only stores $0$ as a single bit into the copy of the [`Builder{:tact}`][builder]. Returns that copy.
+
+As a single [`Cell{:tact}`][cell] can store up to $4$ references, attempts to store more throw an exception with [exit code 8](/book/exit-codes#8): `Cell overflow`.
+
+Usage example:
+
+```tact
+let b: Builder = beginCell();
+let fizz: Builder = b
+    .storeMaybeRef(emptyCell()) // 1, then empty cell
+    .storeMaybeRef(null);       // 0
```

Comment:
what do `1` and `0` mean here?

Reply:
Single `1` and `0` bits

Reply:
this looks confusing to me, I'd either add a TL-B description or removed the comments altogether

Reply:
Elaborated :)